### PR TITLE
FBISCC-3: Generate hof app

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
* Use hof-generator
* Remove travis file created by hof-generator
* Update npm script to run with `npm run dev` rather than `npm run start:dev`. Separate commit to be clear.  I didn't want to wait to submit another PR for this but can do if you want

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM quay.io/ukhomeofficedigital/nodejs-base:v6
```
Comment (markwimpory) [inline Dockerfile:1]: Ancient....

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+language: node_js
```
Comment (markwimpory) [inline .travis.yml]: Delete

File: apps/fbis-ccf/translations/src/en/journey.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "header": "My New HOF Service",
```
Comment (markwimpory) [inline apps/fbis-ccf/translations/src/en/journey.json:2]: Make it FBIS

**Review by markwimpory — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (sulthan-ahmed) [reply Dockerfile:1]: don't you think it's a little out of scope? but ok I'll update

File: apps/fbis-ccf/translations/src/en/journey.json (hunk 3 above)
Comment (sulthan-ahmed) [reply apps/fbis-ccf/translations/src/en/journey.json]: thanks updated :)

**Comment by sulthan-ahmed** [comment]
Thanks @markwimpory

**Pull request merged**